org.jacoco.doc/docroot/doc/license.html

```diff
@@ -275,6 +275,14 @@
       of your accepting any such warranty or additional liability.
 </pre>
 
+<h3>args4j, Version 2.0.28</h3>
+
+<p>
+  <a href="http://args4j.kohsuke.org/">args4j</a> is subject to the terms and
+  conditions of
+  <a href="http://www.opensource.org/licenses/mit-license.php">the MIT license</a>. 
```

Comment:
@Godin For the others I took the original license that comes with the products. Args4j does not come with a license, only with a link to he MIT license. That's why. Let's clarify this in a separate issue?